requirements-mysql.txt

```diff
@@ -1 +1,2 @@
-mysqlclient
+# Packages required to run this project with MySQL as the DB
+mysqlclient==1.4.6
```

Comment:
These `.txt` requirements files are now hand-written. So adding in some commentary to highlight it.